start-console.sh

```diff
@@ -30,13 +30,13 @@ echo "Console URL: http://localhost:${CONSOLE_PORT}"
 if [ -x "$(command -v podman)" ]; then
     if [ "$(uname -s)" = "Linux" ]; then
         # Use host networking on Linux since host.containers.internal is unreachable in some environments.
-        BRIDGE_PLUGINS="${npm_package_consolePlugin_name}=http://localhost:9001"
+        BRIDGE_PLUGINS="${npm_package_consolePlugin_name}=http://localhost:9001,monitoring-plugin=http://localhost:9002"
         podman run --pull=always --rm --network=host --env-file <(set | grep BRIDGE) $CONSOLE_IMAGE
     else
-        BRIDGE_PLUGINS="${npm_package_consolePlugin_name}=http://host.containers.internal:9001"
+        BRIDGE_PLUGINS="${npm_package_consolePlugin_name}=http://host.containers.internal:9001,monitoring-plugin=http://host.containers.internal:9001"
         podman run --pull=always --rm -p "$CONSOLE_PORT":9000 --env-file <(set | grep BRIDGE) $CONSOLE_IMAGE
     fi
 else
-    BRIDGE_PLUGINS="${npm_package_consolePlugin_name}=http://host.docker.internal:9001"
+    BRIDGE_PLUGINS="${npm_package_consolePlugin_name}=http://host.docker.internal:9001,monitoring-plugin=http://host.docker.internal:9002"
     docker run --pull=always --rm -p "$CONSOLE_PORT":9000 --env-file <(set | grep BRIDGE) $CONSOLE_IMAGE
```

Comment:
why do we want to add those changes to start console to our repo? - its only for the observe plugin, not Prometheus , we will still get the data for graphs in OUR plugin with our observe

Reply:
This is here accidentally.